docs/docs/consumer.mdx

```diff
@@ -107,25 +104,26 @@ with connect_with_secret(host="ewb.zepben.com", rpc_port=443, ca=ca, key=key, ce
 
 ### Authentication
 
-Password Credentials and Client credentials OAuth2 flows are supported through the username/password and client_id/client_secret parameters:
+Password Credentials and Client credentials OAuth2 flows are supported through the `connect_with_secret` and `connect_with_password` functions respectively:
 
 ```python
 from zepben.evolve import connect_with_password, connect_with_secret, SyncNetworkConsumerClient
 
-# Password credentials configuration - client_secret optional.
-with connect_with_password(host="ewb.zepben.com", rpc_port=443, conf_address="https://ewb.zepben.com/ewb/auth", secure=True, client_id="some_client_id",
-             username="user@email.com", password="password1") as channel:
+# Client credentials configuration
+async with connect_with_secret(client_id="some_client_id", client_secret="some_client_secret", conf_address="https://ewb.zepben.com/ewb/auth",
+                               host="ewb.zepben.com", rpc_port=443) as channel:
     client = SyncNetworkConsumerClient(channel)
     # ...
 
-# Client credentials configuration
-with connect_with_secret(host="ewb.zepben.com", rpc_port=443, conf_address="https://ewb.zepben.com/ewb/auth", secure=True, client_id="some_client_id",
-             client_secret="some_client_secret") as channel:
+# Password credentials configuration
```

Comment:
Is using the password grant with client secret a frequent use case?

Reply:
No. Not a use case.